### PR TITLE
Research links...

### PR DESCRIPTION
**Pull request by ultrasaurus**
Here's a starter list of intro reading.  
I'd like it to be a bit more balanced before linking to the main site.

### CODE REVIEW

File: resources/intro-reading.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
```
Comment (iisa) [inline resources/intro-reading.md]: Question: do we have a blurb at the top? or a link to this page somewhere? or do we want to just start this collection then add the other stuff later?

Comment (ultrasaurus) [reply resources/intro-reading.md]: yeah, I was thinking to start as a page that wasn't linked from anywhere, then when we feel like the content is good, we can link

agree will need some intro blurb, prolly will end up with groupings

Comment (ilen) [reply resources/intro-reading.md]: What do you think about adding links to non-reading resources (i.e. https://implicit.harvard.edu/implicit/takeatest.html)

Comment (ultrasaurus) [reply resources/intro-reading.md]: I like that idea! added issue to collect ideas/suggestions of what should be added before publishing https://github.com/bridgefoundry/bridgefoundry.github.io/issues/128

**Comment by ultrasaurus** [comment]
starter page for https://github.com/bridgefoundry/bridgefoundry.github.io/issues/128

**Pull request merged**